src/components/BillCategoryTag/BillCategoryTag.svelte

```diff
@@ -0,0 +1,16 @@
+<script lang="ts">
+	import { Tag } from 'carbon-components-svelte';
+
+	export let isLarge = false;
+	let label = 'ขนส่งสาธารณะ';
+	let tagStyle = 'text-text-10 border-solid bg-white m-0 whitespace-nowrap';
```

Comment:
Since `tagStyle` won't change at all, I suggest putting these classes in the Tag's `class` directly.